src/vulnerability-report.ts

```diff
@@ -71,4 +71,18 @@ export class VulnerabilityReport extends K8sApi.KubeObject {
         },
         vulnerabilities: Vulnerability[];
     }
+
+    getImageRef(): string {
+        const {artifact, registry} = this.report
+        let imageId
+        if (artifact.tag) {
+            imageId = ":" + artifact.tag
+        }
+        if (artifact.digest) {
+            imageId = "@" + artifact.digest
+        }
```

Comment:
Is it possible for both of these conditions to be true simultaneously?

Reply:
Good point. You can actually specify tag and digest at the same time, e.g. `docker.io/library/nginx:1.16@sha256:abcdef...`, so I added such conditions as well